program/src/instruction.rs

```diff
@@ -1,184 +1,373 @@
+    )]
     InactivateStake,
 
-    /// Withdraw inactive staked tokens from the vault
-    ///
-    /// After a deactivation has gone through the cooldown period and been
-    /// "inactivated", the authority may move the tokens out of the vault.
-    ///
-    /// 0. `[w]` Config account
-    /// 1. `[w]` Stake account
-    /// 2. `[w]` Vault token account
-    /// 3. `[w]` Destination token account
-    /// 4. `[s]` Stake authority
-    /// 5. `[]` Vault authority, PDA with seeds `['token-owner', stake_config]`
-    /// 6. `[]` SPL Token program
-    /// 7.. Extra required accounts for transfer hook
-    ///
-    /// Instruction data: amount of tokens to move
+    /// Withdraw inactive staked tokens from the vault.
+    #[account(
+        0,
+        writable,
+        name = "config",
+        desc = "Stake config account"
+    )]
+    #[account(
+        1,
+        writable,
+        name = "stake",
+        desc = "Stake account"
+    )]
+    #[account(
+        2,
+        signer,
+        name = "stake_authority",
+        desc = "Stake authority"
+    )]
+    #[account(
+        3,
+        name = "vault_authority",
+        desc = "Vault authority"
+    )]
+    #[account(
+        4,
+        writable,
+        name = "vault_token",
+        desc = "Vault token account"
+    )]
+    #[account(
+        5,
+        writable,
+        name = "destination_token",
+        desc = "Destination token account"
+    )]
+    #[account(
+        6,
+        name = "spl_token_program",
+        desc = "SPL Token program"
+    )]
     WithdrawInactiveStake(u64),
 
     /// Harvests holder SOL rewards earned by the given stake account.
-    ///
-    /// NOTE: This mostly replicates the logic in the rewards program. Since the
-    /// staked tokens are all held by this program, stakers need a way to access
-    /// their portion of holder rewards.
-    ///
-    /// This instruction requires that `unclaimed_rewards` be equal to `0` in
-    /// the token vault account. For ease of use, be sure to call the
-    /// `HarvestRewards` on the vault account before this.
-    ///
-    /// 0. `[]` Config account
-    /// 1. `[w]` Stake account
-    /// 2. `[w]` Vault token account
-    /// 3. `[]` Holder rewards account for vault token account
-    /// 4. `[w]` Destination account for withdrawn lamports
-    /// 5. `[s]` Stake authority
-    /// 6. `[]` Vault authority, PDA with seeds `['token-owner', stake_config]`
-    /// 7. `[]` Stake token mint, to get total supply
-    /// 8. `[]` SPL Token program
+    #[account(
+        0,
+        name = "config",
+        desc = "Stake config account"
+    )]
+    #[account(
+        1,
+        writable,
+        name = "stake",
+        desc = "Stake account"
+    )]
+    #[account(
+        2,
+        writable,
+        name = "vault_token",
+        desc = "Vault token account"
+    )]
+    #[account(
+        3,
+        name = "vault_authority",
+        desc = "Vault authority"
+    )]
```

Comment:
Same comment here about the account ordering being mixed up from the original comments.

Reply:
Please see comment above about account ordering.

Reply:
Reverted back to the original order.